### PR TITLE
For issue #9

### PR DESCRIPTION
**Pull request by littlebear-xbz**
Please just merge setup.py

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,24 +20,28 @@ This is a patch for happybase to support kerberos when connect to hbase thrift s
...
 
+The project is base on happybase https://github.com/python-happybase/happybase
+
+
 # requirements
 
 In order to use kerberos, the following packages is required:
 
-1. python3-dev
-2. libkrb5-dev
```
Comment (normal-cock) [inline README.md:28]: I'm not sure why you have deleted `libkrb5-dev`. It is needed to make `pykerberos` work.  And `python3-dev` is needed to support Python3.

File: README.md [hunk 2]
```diff
@@ -20,24 +20,28 @@ This is a patch for happybase to support kerberos when connect to hbase thrift s
...
+1. python-devel
 
-For example, in ubuntu install by the follow commands:
+For example, in redhat install by the follow commands:
```
Comment (normal-cock) [inline README.md:34]: Maybe you could add another example for redhat alone and keep the ubuntu example unchanged.

File: README.md [hunk 3]
```diff
@@ -20,24 +20,28 @@ This is a patch for happybase to support kerberos when connect to hbase thrift s
...
-sudo apt-get install python-dev
-sudo apt-get install python3-dev
-sudo apt-get install libkrb5-dev
+yum install python-devel
 ```
 
 # Installment
 
-`pip install -U git+https://github.com/normal-cock/happybase-kerberos-patch.git`
+In China for higher installation speed could use "-i https://pypi.tuna.tsinghua.edu.cn/simple"
```
Comment (normal-cock) [inline README.md:52]: Cool, thanks for you complement.

**Review by normal-cock — CHANGES_REQUESTED** [review]

**Comment by normal-cock** [comment]
By the way, could you submit pull request to `develop` branch? I'll merge it to master individually.

File: README.md (hunk 1 above)
Comment (littlebear-xbz) [reply README.md:28]: On redhat , there is no `python3-dev`, I should keep yours

**Comment by littlebear-xbz** [comment]
I will submit the pull request to develop branch.

**Pull request closed without merging**